src/carousel/components/runtime.tsx

```diff
@@ -128,7 +129,7 @@ export const Runtime: React.FC<IProps> = ({ authoredState, interactiveState, set
           let buttonStyle = {};
           let buttonClass = currentSlide === index ? css.activeButton : "";
           if (interactive.navImageUrl) {
-            buttonStyle = { backgroundImage: "url(" + interactive.navImageUrl + ")" };
+            buttonStyle = { backgroundImage: cssUrlValue(interactive.navImageUrl)};
```

Comment:
It looks like `drag-and-drop` is doing something similar in container.tsx. Did you check to see whether the same fix is appropriate there?

Reply:
Good catch.  I've updated it in this commit: https://github.com/concord-consortium/question-interactives/pull/124/commits/dbe5191b1423fcdaf8fb7c80ecc498a507334f79